nomos-da/network/core/Cargo.toml

```diff
@@ -27,6 +27,7 @@ tracing-subscriber       = "0.3.18"
 
 [dev-dependencies]
 blake2             = "0.10"
+kzgrs              = { workspace = true }
```

Comment:
Who did it work before? :thinking:

Reply:
We used `Default::default()` for commitments, proofs - https://github.com/logos-co/nomos-node/pull/1093/files#diff-c0e266e9e2ccfbc5a190cf49c9e696a1e023484d68add66e74beacd57bf6cb1bR101-R103

Reply:
Ah yeah, makes sense!

Reply:
Exactly. I would argue using concrete types is easier to understand especially for people that do not deal with this specific part of the code that often. But again, if most people prefer the old way, I can revert the changes.

Reply:
No, using the types is more readable. We can keep this yes.